conference.js

```diff
@@ -2423,9 +2446,7 @@ export default {
                 .then(() => {
                     // If audio was muted before, or we unplugged current device
                     // and selected new one, then mute new audio track.
-                    if (audioWasMuted
-                        || currentDevices.audioinput.length
-                        > availableAudioInputDevices.length) {
+                    if (audioWasMuted) {
```

Comment:
I feel like one case where I would want mute on device change is if my current device got unplugged. Maybe I don't want it to automatically switch to another camera. Do you think that's a likely concern?

Reply:
So this covers the situation where you are using device A and you unplug device B, which is not used, you don't want to mute device A.
In the case where you are using device A and it was muted and you unplug it, we start using device B and mute it, keeping the state in the conference. For me this makes sense. 
I don't think switching to an available device is a problem, it is like you have in settings to use camera A, but on loading the app that device is missing we start using camera B. For me, both cases are almost identical. Is this ok?

Reply:
I want to be in control of what camera gets selected or have some kind of preview saying that "this camera is gonna be used and it looks like this." As a user, and someone that keeps the laptop camera covered with electrical tape, I'm pretty picky about camera usage. That said, it is true that the use case is quite similar so it's hard for me to argue that you should change precedent with this pull request, so I'm okay with it.